packages/cli/src/commands/publishers.ts

```diff
@@ -157,16 +183,16 @@ export async function publishers({ cliSettings, options, packageRef }: Params) {
   );
   console.log();
 
-  const verification = await prompts({
-    type: 'confirm',
-    name: 'confirmation',
-    message: 'Proceed?',
-    initial: true,
-  });
+  if (!options.skipConfirm) {
+    const confirm = await prompts({
+      type: 'confirm',
+      name: 'confirmation',
+      message: 'Proceed?',
+    });
 
-  if (!verification.confirmation) {
-    console.log('Cancelled');
-    process.exit(1);
+    if (!confirm.confirmation) {
+      process.exit(0);
```

Comment:
The error code should be different to `0`, to mark the script as not completed or intervened by the user